evmrpc/config.go

```diff
@@ -75,6 +75,9 @@ type Config struct {
 	// controls whether to synchronously flush receipts before block finalze or not
 	FlushReceiptSync bool `mapstructure:"flush_receipt_sync"`
 
+	// DisableWatermark disables cross-store watermark gating (intended for tests)
```

Comment:
Why tests need to disable watermark? Shouldn't we fix the test and make sure new watermark works for all tests?